sale_commission_oca/views/sale_order_view.xml

```diff
@@ -50,11 +50,11 @@
         </field>
     </record>
     <!-- This view is needed for proper visualization of the popup / embedded one2many -->
-    <record model="ir.ui.view" id="view_sale_order_line_tree">
-        <field name="name">sale.order.line.agent.tree</field>
+    <record model="ir.ui.view" id="view_sale_order_line_list">
```

Comment:
Please don't change XML-IDs for not over-complicating inheritance.

Reply:
@pedrobaeza done :)